docs/Job-DSL-Commands.md

```diff
@@ -6,31 +6,27 @@ The DSL execution engine exposes a method, called `job`. This `job` method impli
 and the closure to this method can be used to define the job's settings. The only mandatory option is `name`.
```

Comment:
Yes, thanks for proofreading. It has been adapted as part of a follow-up pull request, see https://github.com/jenkinsci/job-dsl-plugin/blob/master/docs/Job-DSL-Commands.md.